### ISSUE TITLE
Support for Composite Primary Keys in Conflict Resolution

### ISSUE THREAD

**Issue by nicolazreinh** [Mimetis/Dotmim.Sync#1077]
I encountered an issue related to conflict resolution while working with two clients connected and synced with the same server database. The problem arises from using a table with a composite primary key, which consists of an incremental integer and a datetime.

Scenario:

On the first client, I added an entity to the table, and it received an auto assigned ID as the integer component of the composite primary key.
Subsequently, I added another entity on the second client, which also utilized the same integer component for the primary key.
Upon performing the synchronization process, a conflict occurred on the second client, and it resulted in the override of the values.
Expected Outcome:
Ideally, during conflict resolution, the system should handle composite primary keys correctly to prevent data overrides. The conflict resolution mechanism should be able to identify that the entities from both clients have distinct datetime components in their composite primary keys, indicating that they are different records.

**Comment by Mimetis** [comment]
I need a simple repro, to be able to help you i think

**Comment by nicolazreinh** [comment]
Thank you for your quick response. I appreciate your willingness to help. Upon further investigation, I realized that I was missing a crucial step in my initial implementation. The conflict resolution is indeed working as expected, and the system is capable of handling composite primary keys correctly.
I apologize for any confusion caused by my initial report. I have now resolved the issue on my end by ensuring that the datetime component of the composite primary key is also taken into account when identifying conflicts during synchronization.
Thank you again for your time and assistance in this matter.
